lib/models/article.js

```diff
@@ -498,4 +498,33 @@ Object.defineProperty(Article.prototype, 'isWeekend', {
 	}
 });
 
+Object.defineProperty(Article.prototype, 'contentClassification', {
+	get : function(){
+		if(this._contentClassification){
+			return this._contentClassification;
+		}
+
+		var results = /cms\/s\/([0-3])\//i.exec(this.raw.item.location.uri);
```

Comment:
also just so you know we've raised this as an issue to the capi team as this is missing from capi v2